api/v1alpha3/common_types.go

```diff
@@ -22,9 +22,9 @@ import (
 )
 
 const (
-	// ClusterLabelName is the label set on machines linked to a cluster and
+	// ClusterNameLabel is the label set on machines linked to a cluster and
```

Comment:
Please only rename the constants in the latest apiVersion (v1beta1, not v1alpha3/v1alpha4)

Reply:
Understood, will update the code to rename only for v1beta1.

Q) What about the usage of constant `ClusterLabelName` in files inside `bootstrap/kubeadm/internal`, `cmd/clusterctl`, `controller/external`, `controlplane/kubeadm/internal` etc. Basically every other file apart from `api/*`. I have changed it for now, but is it required to do so?

Reply:
Yes it is and that's fine

Reply:
Regarding the usage of v1alpha3/v1alpha4, We are using some types and structs from `v1alpha3/v1alpha4` but no constants.

Reply:
Took a look those usages of v1alpha3/v1alpha4 are fine

Reply:
Changes lgtm to so far